dallinger/experiment.py

```diff
@@ -449,21 +450,35 @@ def run(self, exp_config=None, app_id=None):
             app_id = str(uuid.uuid4())
 
         self.app_id = app_id
-        self.exp_config = exp_config
+        self.exp_config = exp_config or {}
 
-        dlgr.command_line.deploy_sandbox_shared_setup(
-            app=app_id,
-            verbose=self.verbose,
-            exp_config=exp_config
-        )
+        bot = kwargs.get('bot')
+        if bot:
+            del kwargs['bot']
+            kwargs['recruiter'] = 'bots'
```

Comment:
Why not add bot=False to the `run` function signature before `**kwargs`, and then this becomes simply:
```
if bot:
    kwargs['recruiter'] = 'bots'
```